modules/client-configuration/pages/client-upgrades-major.adoc

```diff
@@ -135,3 +130,16 @@ When the client next boots, it will use grub to boot the new kernel with its ini
 PXE booting is not used during this process.
 
 Approximately three minutes after the job was fetched, the client will go down for reboot.
+
+
+ifeval::[{suma-content} == true]
+[NOTE]
+====
+For Salt clients, use the ``spacewalk/minion_script`` snippet to register the client again after migration has completed.
+====
+endif::[]
+ifeval::[{uyuni-content} == true]
+// Uyuni is Salt clients only.  Uyuni does not manage traditional clients.
```

Comment:
????

Uyuni contains the traditional stack

https://build.opensuse.org/project/show/systemsmanagement:Uyuni:Stable:SLE12-Uyuni-Client-Tools

https://build.opensuse.org/project/show/systemsmanagement:Uyuni:Stable:SLE15-Uyuni-Client-Tools